ADOPTERS.md

```
@@ -17,6 +17,7 @@
  | ---- | ---- | ----------- | ------- |
  | <!-- ![Acme Logo](path/to/logo.png) --> | **Example Acme Corp** | Uses Nettacker for automated penetration testing. | https://acme.example.com |
  | <!-- ![SecurityCo Logo](path/to/securityco-logo.png) --> | **Example SecurityCo** | Integrates Nettacker into their CI/CD pipeline for continuous security assessment. | https://securityco.example.org |
+ | HISC (Information Security Club - HCMUTE) | Using Nettacker for automated reconnaissance, vulnerability research, and security training in CTF environments. | https://phucquan.github.io/ |
```

markdownlint-cli2 (0.21.0)

[warning] Table column count
Expected: 4; Actual: 3; Too few cells, row will be missing data

(MD056, table-column-count)